src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -621,6 +704,39 @@ void StartDefaultTask(void *argument)
     /* USER CODE END 5 */
 }
 
+/* USER CODE BEGIN Header_test_msg_update */
+/**
+ * @brief Function implementing the testMsgUpdate thread.
+ * @param argument: Not used
+ * @retval None
+ */
+/* USER CODE END Header_test_msg_update */
+void test_msg_update(void *argument)
+{
+    /* USER CODE BEGIN test_msg_update */
+
+    // TODO this is a placeholder task to test sending robot status NOT
+    // associated with a ticket because how the robot status msgs will be
+    // updated and sent is TBD
```

Comment:
```suggestion
    // updated and sent is https://github.com/UBC-Thunderbots/Software/issues/1518
```

Reply:
This includes updating all the sensor, temperature component messages, we don't know when/how we will receive that data. 

I added 1518 near the `sys_now` call, it doesn't make sense here?

Reply:
oh mb

Reply:
offline discussion: added a placeholder ticket